tests/steps/steps.py

```diff
@@ -39,6 +39,7 @@ def parse_string(text):
 
 
 token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+indexer_port = 59999
```

Comment:
nit: if unused, can we delete?

Reply:
this one is used in `other_v2_steps.py`

Reply:
Gotcha, thanks - we may have to delete the v1 `steps` so might have to port these over in the future, but looks fine as-is.